### PR TITLE
Position field documentation 

### PR DESCRIPTION
**Pull request by lfbrock**

### CODE REVIEW

File: source/help/settings/account-settings.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Usernames are unique identifers appearing next to all posts. Pick something easy
 #### Nickname
 Nicknames appear in the Direct Messages member list and team management modal. You will not receive mention notifications when someone types your nickname unless you add it to the *Words That Trigger Mentions* in **Account Settings > Notifications**.
 
+#### Position
+Position can be used to describe your role or job title. It appears in the profile popup that shows up when you click a user's name in the center channel.
```
Comment (jasonblais) [inline source/help/settings/account-settings.md]: *center channel or right-hand sidebar

Comment (lfbrock) [inline source/help/settings/account-settings.md]: Hmm yeah, I actually considered adding that but wasn't sure what our current stance was on clarity over completeness. 

In this case I'll add it, it doesn't matter too much here, but I think there's a lot of places in our docs where we focus on completeness over clarity/usefulness to the end user.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -840,6 +840,15 @@ Nickname Attribute
 | This feature's ``config.json`` setting is ``"NicknameAttribute": ""`` with string input.                                                                             |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Position Attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(Optional) The attribute in the AD/LDAP server that will be used to populate the position field in Mattermost (typically used to describe a person's job title or role at the company). When set, users will not be able to edit their position, since it is synchronized with the LDAP server. When left blank, users can set their own Position in Account Settings.
```
Comment (JeffSchering) [inline source/administration/config-settings.rst]: "When blank, users can set their own position in the **Account Settings** dialog."

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -1036,6 +1045,15 @@ Nickname Attribute
 | This feature's ``config.json`` setting is ``"NicknameAttribute": ""`` with string input.                                                                             |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Position Attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(Optional) The attribute in the SAML Assertion that will be used to populate the position field for users in Mattermost.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: an extra period after Mattermost

File: source/help/settings/account-settings.md [hunk 4]
```diff
@@ -27,7 +30,7 @@ Settings to configure your password, view access history, and view or logout of
 You may change your password if you’ve logged in by email. If you are signing in using a single sign-on service, the password field is not editable, and you must access your SSO service’s account settings to update your password.
 
 #### Multi-Factor Authentication 
-When this option is available you can requires a phone-based passcode in addition your password for sign-in. 
+When this option is available you can require a phone-based passcode in addition your password for sign-in.
```
Comment (jasonblais) [inline source/help/settings/account-settings.md]: `in addition` --> `in addition to`

**Comment by esethna** [comment]
Pushed minor tweaks to address other comments

**Pull request merged**